io/zenoh-transport/tests/endpoints.rs

```diff
@@ -334,7 +334,7 @@ AXVFFIgCSluyrolaD6CWD9MqOex4YOfJR2bNxI7lFvuK4AwjyUJzT1U1HXib17mM
 #[cfg(feature = "transport_quic")]
 #[tokio::test(flavor = "multi_thread", worker_threads = 4)]
 async fn endpoint_quic() {
-    use zenoh_link::quic::config::*;
+    use zenoh_link::tls::config::*;
```

Comment:
Here we have a conflict for `features`. This test runs for `#[cfg(feature = "transport_quic")]` bug it actually requires #[cfg(feature = "transport_tls")] as well for `use zenoh_link::tls::config::*;`. This means we can never compile `transport_quic` without activating `transport_tls` feature.